src/app/core/pipes/duration-as-days.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import * as moment from 'moment';
+
+@Pipe({name: 'appDurationAsDays'})
+export class DurationAsDaysPipe implements PipeTransform {
+    transform(duration: moment.Duration) {
+        if (!duration) {
+            return undefined;
+        }
+        return this.ceilToFirstToHalf(moment.duration(duration).asDays());
+    }
+
+    private ceilToFirstToHalf(value) {
```

Comment:
Wouldn't 
`return Math.round(value*2)/2;`
work?

Reply:
for example: 1.10*2/2 = 1.1

Reply:
No it's not.

Reply:
you are right, but I still want to ceil (going up) to the nearest 0.5 so a bigger precision is needed indeed not just first decimal.